src/start/RealmInputScreen.js

```diff
@@ -59,20 +60,21 @@ export default function RealmInputScreen(props: Props): Node {
   );
 
   const tryRealm = React.useCallback(async () => {
-    const parsedRealm = urlFromInputValue(realmInputValue);
-    if (!parsedRealm) {
+    if (!urlFromInputValue) {
```

Comment:
The "from" in this name means, to me, that it should be a function that returns a URL and takes the/an "input value" as an argument from which to compute the URL.

If you grep for similar names in our codebase, the bulk of them follow that same pattern: `git grep '\w+From\w+' src/`, or `git grep -oh '\w+From\w+' src/ | sort | uniq -c` for a summary.

So it's confusing to see it here mean instead the URL itself, rather than a function that computes one.

It looks like this gets further revised in a later commit anyway. So perhaps simplest is to let it keep the same name that this existing local had, `parsedRealm`; then the later commit does what it wants from there.

Reply:
Hmm. I recommend getting a better distribution of Git. :slightly_smiling_face: 

Works fine on my Mac, as well as on my usual Linux desktop.

On that Mac, my `git` is a Git 2.31.1 from Homebrew. Perhaps `brew install git`?

Ah indeed, and if I use `/usr/bin/git` there (to get the Git supplied with the OS), I get the same error as you do.

Reply:
Here's my `git --version`:

```
$ git --version
git version 2.37.1 (Apple Git-137.1)
```

> Perhaps `brew install git`?

Sure, I'll try that.

Reply:
OK, now

```
$ git --version
git version 2.38.1
```

and `it grep -oh '\w+From\w+' src/ | sort | uniq -c` gives results with no error now! 🎉 Thanks.